src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -43,7 +45,7 @@ const GovernanceSection = () => {
                   {claimingApp ? (
                     <SafeAppsErrorBoundary render={() => <SafeAppsLoadError onBackToApps={() => {}} />}>
                       <AppFrame
-                        appUrl={`${claimingApp.url}${SNAPSHOT_WIDGET_ID}`}
+                        appUrl={`${claimingApp.url}${SNAPSHOT_WIDGET_ID}${isDarkMode ? '+dark' : ''}`}
```

Comment:
I would recommend we sooner pass this as a search parameter, otherwise it is part of the `hash`.

Reply:
I disagree with this. Widget if is passed in the hash, why would dark mode be passed differently?

Reply:
Besides, a search param invalidates the HTTP cache. So you’d have to download that app again when the theme changes.

But most importantly I just don’t get why we come up with arbitrary requests to change the code like that.